.github/dependabot.yml

```diff
@@ -6,3 +6,11 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+  - package-ecosystem: docker
```

Comment:
I'm not sure if I like the idea of dependabot updating our image. I'll like @bcrochet and @komish chime in.

Reply:
For example, it'll make PRs like: https://github.com/test-network-function/cnf-certification-test/pull/1118

Reply:
Yeah, we used `latest` for UBI and our builder is golang `1.20` at this point. I don't want it updating the builder before we've updated the codebase, and I also don't want the bot updating to `minor` versions of the builder. Especially, since we had an issue last week with the version of `libc` being updated on the base `1.20` version.

Reply:
I removed the `Docker` section from the config.  The PR only does github actions updates now.